java/vertx4/config.yaml

```
@@ -0,0 +1,14 @@
+framework:
+  website: https://vertx.io/docs/vertx-core/java/
+  version: 4.0.0
```

I prefer `X.Y` format
```suggestion
  version: 4.0
```